### ISSUE TITLE
GPG key error when packaging

### ISSUE THREAD

**Issue by ChameleonScales** [matze/inkscape-flatpak#3]
I'm trying to package the flatpack from the repo but I'm not very good with gpg keys I guess.
I use Kgpg and entered the ID of the only key I have but got this error when doing "make release" :
```
flatpak-builder --force-clean --repo=repo --ccache --gpg-sign=<(my key ID)> inkscape org.inkscape.Inkscape.json
/bin/sh: (my key ID) : No such file or directory
Makefile:4: recipe for target 'release' failed
make: *** [release] Error 1
```
What am I doing wrong ?

**Comment by matze** [comment]
So, you have created a `Makefile.config` that contains the `GPG_KEY` variable with your key id? It looks like a long hexstring.

By the way, the package is on flathub now, so you don't need to build anything yourself.

**Comment by ChameleonScales** [comment]
Yes I did all of this. Didn't know about the flathub repo, I will try this. But is it always as up to date as the github repo ?

**Comment by ChameleonScales** [comment]
not working either, when entering :

> flatpak remote-ls --app flathub

I get :

> error: GPG verification enabled, but no summary signatures found (use gpg-verify-summary=false in remote config to disable)

I will have a look into it when I have time. It has nothing to do with the Inkscape flatpak.
